cmd/createGithub.go

```diff
@@ -53,29 +42,39 @@ var createGithubCmd = &cobra.Command{
 			log.Panic(err)
 		}
 
+		progressPrinter.GetInstance()
+		progressPrinter.SetupProgress(4, globalFlags.SilentMode)
+
+		infoCmd.Run(cmd, args)
+		var kPortForwardArgocd *exec.Cmd
+		progressPrinter.AddTracker("step-0", "Process Parameters", 1)
+		progressPrinter.AddTracker("step-github", "Setup gitops on github", 3)
+		progressPrinter.AddTracker("step-base", "Setup base cluster", 2)
+		progressPrinter.AddTracker("step-apps", "Install apps to cluster", 6)
+
 		progressPrinter.IncrementTracker("step-0", 1)
 
 		if !globalFlags.UseTelemetry {
-			informUser("Telemetry Disabled")
+			informUser("Telemetry Disabled", false)
 		}
-		informUser("Creating gitops/metaphor repos")
+		informUser("Creating gitops/metaphor repos", false)
 		err = githubAddCmd.RunE(cmd, args)
 		if err != nil {
 			log.Println("Error running:", githubAddCmd.Name())
 			return err
 		}
-		informUser("Create Github Repos")
+		informUser("Create Github Repos", false)
 		progressPrinter.IncrementTracker("step-github", 1)
 		err = loadTemplateCmd.RunE(cmd, args)
```

Comment:
if we use a viper flag, you don't need to change the signature.

Reply:
I would like to avoid adding more content to the kubefirst file , and having silent mode flag in the viper file doesn't sound good IMHO